index.html

```diff
@@ -83,6 +86,119 @@ <h3>Ignite</h3>
 
         </div>
         <p id="wholeProgram">SEE THE WHOLE PROGRAM</p>
+        <button type="button" id="programButton">SEE THE WHOLE PROGRAM</button>
+    </div>
+
+    <div class="speakers">
+        <h2>Featured Speakers</h2>
+    </div>
+    <div class="underline"></div>
+    
+    <div class="divSpeaker">
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Elmar Abdulkarimov</h2>
+                <h5>Project Manager</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Emil Aliyev</h2>
+                <h5>Finance Manager</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        
+        
+    </div>
+    <div class="divSpeaker">
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Alina Abdul</h2>
+                <h5>CEO</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Matanat Rahimova</h2>
+                <h5>Head of HR</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        
+        
+    </div>
+    <div class="divSpeaker">
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Maryam Kerimova</h2>
+                <h5>Head of Finance</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        <div class="speakerInfo">
+            <img src="./images/lecture.png" alt="lecture">
+            <div class="ddd">
+                <h2>Zuzu Rehimli</h2>
+                <h5>Marketing director</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, ut hic. Ratione itaque deserunt corrupti</p>
+        
+            </div>
+        </div>
+        
+        
+    </div>
```

Comment:
- [ ]  **As a part of the project requirements, the entire featured speaker section has to be dynamically generated using Javascript. Please implement it.**

Following is the requirement in project guidelines. 


**Dynamic page**

The section "Featured speakers" should be created dynamically in JavaScript.
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

Reply:
Done -commit "Create dynamic section"